Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -425,7 +431,7 @@ class HorizontalFlowLayout: UICollectionViewLayout {
         cachedAttributes = nil
         // Sometimes when the topsiteCell isnt on the screen the newbounds that it tries to layout in is very tiny
         // Resulting in incorrect layouts. So only layout when the width is greater than 320.
-        return newBounds.width >= 320
+        return newBounds.width > 0
```

Comment:
Wouldn't this always be `true`? Also, the comment above this line needs updated to reflect the new logic.